util/src/main/java/org/datacommons/util/ComplexValueParser.java

```diff
@@ -0,0 +1,335 @@
+package org.datacommons.util;
+
+import static org.datacommons.proto.Mcf.ValueType.*;
+import static org.datacommons.util.McfParser.splitAndStripWithQuoteEscape;
```

Comment:
i think we should avoid imports of individual methods - it makes it harder to figure out, while reading, where the definition is situated. since this is in the same package, can this be used with McfParser.splitAndStripWithQuoteEscape(...) ?

Reply:
I see.  The advanced IDE technology auto-added all the imports, and I wasn't paying attention as long as things compiled.  I've fixed this one and others in tests manually.